src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4537,15 +4524,16 @@ public override BoundNode VisitIncrementOperator(BoundIncrementOperator node)
 
         public override BoundNode VisitCompoundAssignmentOperator(BoundCompoundAssignmentOperator node)
         {
-            VisitLvalue(node.Left); // https://github.com/dotnet/roslyn/issues/29962 Method should be called VisitValue rather than VisitLvalue.
-            TypeSymbolWithAnnotations leftType = _resultType;
+            Visit(node.Left); // https://github.com/dotnet/roslyn/issues/29962 Method should be called VisitValue rather than VisitLvalue.
```

Comment:
>// https://github.com/dotnet/roslyn/issues/29962 Method should be called VisitValue rather than VisitLvalue. [](start = 30, length = 108)

Comment can be removed.